ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/ChangeId.scala

```diff
@@ -3,14 +3,31 @@
 
 package com.daml.ledger.participant.state.v2
 
+import com.daml.lf.crypto.Hash
 import com.daml.lf.data.Ref
 
 /** Identifier for ledger changes used by command deduplication
```

Comment:
```suggestion
/** Identifier for ledger changes used by command deduplication.
```

Reply:
(Not really about this PR but since you're here)

Reply:
Changed.